pages/common/peerflix.md

```diff
@@ -0,0 +1,19 @@
+# peerflix
+
+> Stream torrents to your favorite player.
+
+- Stream a torrent:
+
+`peerflix "{{torrent url|magnet link}}"`
+
+- Stream a torrent to a specific player:
+
+`peerflix "{{torrent url|magnet link}}" --{{player}}`
+
+- Stream a torrent to VLC:
+
+`peerflix "{{torrent url|magnet link}}" --vlc`
+
+- Stream music from a torrent:
```

Comment:
How is this different to the above example? What does `-a` do (and is there a long form of it?)?

Reply:
-a stands for all (all files in the torrent), I did not even know the long form was implemented ;) I will change that right away

Reply:
Thanks :D

Reply:
Please update the description to mention that this will stream all files in the torrent.